src/Files.App/Utils/Logger/SentryLogger.cs

```diff
@@ -23,15 +23,19 @@ public bool IsEnabled(LogLevel logLevel)
 		public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
 		{
 			// Unhandled exceptions are captured in AppLifecycleHelper.HandleAppUnhandledException
-			if (exception is null || exception.Data[Mechanism.HandledKey] is false)
+			if
+			(
+				exception is null ||
+				exception.Data[Mechanism.HandledKey] is false ||
+				logLevel == LogLevel.Information
```

Comment:
```suggestion
				logLevel <= LogLevel.Information
```
Trace and Debug should be filtered out as well.